Model/DownloadService.swift

```diff
@@ -23,16 +23,15 @@ class DownloadService: NSObject, URLSessionDelegate, URLSessionTaskDelegate, URL
         configuration.allowsCellularAccess = true
         configuration.isDiscretionary = false
         configuration.sessionSendsLaunchEvents = true
-        configuration.timeoutIntervalForResource = 300
```

Comment:
This time out is removed, because `timeoutIntervalForResource` is for the timeout period for the whole download task, and downloads usually takes way more than 5 mins to complete.

`timeoutIntervalForRequest` is the correct one to use, but when set I never get call back from `URLSession` even when I enable flight mode. So I am not sure there is a way to do timeout without implementing it myself